java/src/jmri/jmrix/can/cbus/node/CbusNodeConstants.java

```diff
@@ -504,8 +504,8 @@ private static Map<Integer, String> createExtra44Map() {
         result.put(3, "CAN SPROG programmer/command station.");
         result.put(4, "System booster");
         result.put(5, "Unsuppoerted module type");
-        result.put(6, "CBUS I/O module.");
-        result.put(7, "Isolated USB to CAN interface with CBUS node.");
+        result.put(6, "Isolated USB to CAN interface with CBUS node.");
+        result.put(7, "CBUS I/O module.");
```

Comment:
Just checking that this change is intended, since I don't know the meaning of the digits 6 and 7.